src/App.js

```diff
@@ -1,25 +1,42 @@
-import logo from './logo.svg';
 import './App.css';
+import { Header } from './components/Header';
+import { Sidebar } from './components/Sidebar';
+import { Home } from './pages/Home';
+import { VideoPage } from './pages/VideoPage';
```

Comment:
You can have a common index.js file to reduce import lines here from components and pages